subsys/net/openthread/Kconfig.defconfig

```diff
@@ -10,6 +10,18 @@
 
 if NET_L2_OPENTHREAD
 
+if OPENTHREAD_THREAD_VERSION_1_3
+
+choice OPENTHREAD_IMPLEMENTATION
+	default OPENTHREAD_LIBRARY if BOARD_NRF52840DONGLE_NRF52840
```

Comment:
I'd rather make this depending on the SoC instead of the DK. I.e.: `CONFIG_SOC_NRF52840`.